src/api/libcellml/units.h

```diff
@@ -400,6 +400,33 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     static double scalingFactor(const UnitsPtr &units1, const UnitsPtr &units2);
 
+    /**
+     * @brief Return a bool which indicates if the units are loosely comparable.
+     * 
+     * Two units are loosely comparable if they share the same units ONLY (kg,s,m,...),
+     * this means we are uninterested in unit multipliers. 
+     *
+     * @param1 units1 The first units to compare.
+     * @param2 units2 The second units to compare.
+     *
+     * @return bool indicating whether units are comparable (true), false otherwise.
+     */
+    static bool isEquivalentTo(const UnitsPtr &units1, const UnitsPtr &units2);
```

Comment:
`isEquivalentTo` doesn't seem appropriate for a static method. I'd go with `areEquivalent` or maybe just `equivalent`.

Reply:
I would go with `equivalent`.